lib/prism/pack.rb

```diff
@@ -148,6 +148,8 @@ def describe
             end
           when LENGTH_MAX
             base + ", as many as possible"
+          else
+            raise
```

Comment:
my read from pack.c is that INTEGER type would never return anything but `LENGTH_FIXED` or `LENGTH_MAX`, but I couldn't model that dependent enum structure in RBS. So added this `else` case to satisfy steep's flow analysis.

Reply:
Agreed.